main/attitude.h

```diff
@@ -60,7 +60,7 @@ prior to launch. Once launch is detected we will 'lock in' those chunks and cont
 recording changes forever.
 **************************************************************************************/
 Directional getAttitude(Directional gyro, bool hasLaunched){//only calibrated gyro data should go in here
-  const int hz = 100; //number of readings per second
+  static constexpr int hz = 100; //number of readings per second
```

Comment:
This line should be moved to the top of the file for additional visibility. In the future we might move it out of the file entirely. Also, please change the variable name to something more descriptive than `hz`. maybe `gyroSampleRateHz`.